src/components/Sections/SectionChart/SectionPieChart.js

```diff
@@ -129,22 +129,24 @@ const SectionPieChart = ({ data, style, dimensions, legend, chartProperties = {}
                 }
               </Pie>
               <Tooltip />
-              <Legend
-                wrapperStyle={chartProperties.layout === CHART_LAYOUT_TYPE.vertical ? {
-                  width: '100%'
-                } : { top: 10 }}
-                content={
-                  <ChartLegend
-                    iconType="square"
-                    capitalize={legendStyle.capitalize === undefined || legendStyle.capitalize}
-                    data={preparedData}
-                    formatter={formatValue}
-                    height={legendHeight}
-                    style={legendStyle && legendStyle.style}
-                  />
-                }
-                {...legendStyle}
-              />
+              {preparedData.length > 0 && legendStyle && !legendStyle.hideLegend &&
```

Comment:
if we don't  have legendStyle
we will not render the legend, 
what about BC ? 
would it brake ?

Reply:
legendStyle object is not new, so it should be there for older versions.
regarding the "hideLegend" Same question applied to bar/line chart, no?

Reply:
the code in bar/line for legendStyle is from day one